src/go/wsl-helper/cmd/root.go

```diff
@@ -27,10 +27,7 @@ var rootCmd = &cobra.Command{
 	Short: "Rancher Desktop WSL2 integration helper",
 	Long:  `This command handles various WSL2 integration tasks for Rancher Desktop.`,
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
-		if viper.GetBool("verbose") {
-			logrus.SetLevel(logrus.TraceLevel)
-		}
-		logrus.SetLevel(logrus.TraceLevel)
+		logrus.SetLevel(logrus.InfoLevel + logrus.Level(viper.GetInt("verbose")))
```

Comment:
Is this conventional? To me it looks like it's relying on `logrus.InfoLevel + viper.GetInt('verbose') == logrus.TraceLevel`. Unless it's typical usage, I would prefer an `if/else` stmt

Reply:
That's for being able to set `--verbose=2` to get trace (`--verbose` by itself is just debug).  `if/else` would have to have multiple levels.